supportedBackends/aws/src/test/scala/cromwell/backend/impl/aws/AwsBatchAsyncBackendJobExecutionActorSpec.scala

```diff
@@ -32,7 +32,6 @@
 package cromwell.backend.impl.aws
 
 import java.util.UUID
-
```

Comment:
Can you back out the whitespace changes to these files that were not otherwise changed?